test/JDBC/input/BABEL-4261.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE t_babel4261 (a int, b int);
+GO
+
+select set_config('parallel_setup_cost', 0, false);
+select set_config('parallel_tuple_cost', 0, false);
+select set_config('min_parallel_table_scan_size', 0, false);
+GO
+
+select a, count(*) from t_babel4261 group by a order by 2; -- should not crash
+GO
+
+DROP TABLE t_babel4261;
```

Comment:
```
select set_config('parallel_setup_cost', 0, false);
select set_config('parallel_tuple_cost', 0, false);
select set_config('min_parallel_table_scan_size', 0, false);

```

could you set those three parameters back to default value in this place

Reply:
I'm not quite sure how to set them back since I don't know what the default value is. Also, it seems that in this test https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/48d416c52d445080022b738bc9f5d5fd7c0b777c/test/JDBC/input/pg_hint_plan/BABEL-3294.sql#L14 they are not set back. So I guess not setting them back is okay?

Reply:
No it's not, 3294 didn't do the right thing in here

```
EXEC sp_babelfish_configure 'babelfishpg_tsql.xxxx', 'default';
```

Would you try this to reset.

Reply:
I tried doing that, but it doesn't seem to work:
```
1> EXEC sp_babelfish_configure 'babelfishpg_tsql.parallel_setup_cost', 'default';
2> go
Msg 33557097, Level 16, State 1, Server BABELFISH, Line 1
unknown configuration: babelfishpg_tsql.parallel_setup_cost
```